index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Product Landing Page</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <a href="#">Explore</a>
+      <a href="#">Start a project</a>
+      <a href="index.html"><img src="logo.png"></a>
+      <input type="text" placeholder="Search">
+      <a href="#">Log in</a>
+    </nav>
+    <h1>The Minimalist’s Wall Calendar | Visualize Your Year</h1>
+    <p>Minimalist, forgiving, and beautiful. A wall calendar designed to make 2021 feel like a breath of fresh air.</p>
+  </header>
+  <main>
+    <img src="hero.png" alt="The calendar in a desk setting">
+    <p>Project we Love | Product Design | Cambridge, MA</p>
+    <aside>
+
+      <p>
+        $13,757<br>
+        pledged of $1,200 goal
+      </p>
+      <p>
+        215 <br>
+        backers
+      </p>
+      <p>
+        19 <br>
+        days to go
+      </p>
+      <button>Back this Project</button><br>
+      <button>Remind me</button>
+      <p>
+        <sub>All or nothing. This project will only be funded if it reaches its goal by Thu, November 26 2020 1:35 PM
+          CST.</sub>
+      </p>
+    </aside>
+    <nav>
+      <a href="#">Campaign</a> | <a href="#">FAQ</a> | <a href="#">Updates<sup>2</sup></a> | <a
+        href="#">Comments<sup>19</sup></a> | <a href="#">Community</a>
+    </nav>
+    <aside>
+      <nav>
+        <a href="#">STORY</a>
+        <a href="#">RISKS</a>
+        <a href="#">ENVIRONMENTAL
+          COMMITMENTS</a>
```

Comment:
This nav is inside of an aside, but it feels directly related to the article below it. Since it's attached to the article and the article would be incomplete without it, it should probably just be inside the article tags.

Having said that, I'm not completely married to that semantic distinction, I just wanted to call it out.